internal/install/application_configuration.go

```diff
@@ -150,6 +158,15 @@ func selectElasticAgentImageName(version string) string {
 		logger.Errorf("stack version not in semver format (value: %s): %v", v, err)
 		return elasticAgentImageName
 	}
+
+	disableWolfiImages := false
+	valueEnv, ok := os.LookupEnv(disableElasticAgentWolfiEnvVar)
+	if ok && strings.ToLower(valueEnv) != "false" {
```

Comment:
Perhaps we should default to true till the image is available if we want to merge this. Wdyt?

Reply:
Ok, I'll push the changes to revert the condition.
Once the image is available I'll create a follow-up PR.